vuepress/docs/next/docs/curate/bundle-details.md

```diff
@@ -36,16 +36,17 @@ The docker-based approach is an improvement of the previous Entando Bundle struc
     mfe2/   
     mfe3/   
     mfe3-config        
-  platform/     <= platform specific components    
-    pageModels/
-      pageModel.yaml
+platform/     <= platform specific components    
+    pageTemplates/
+      page-template.yaml
     pages/  
   svc/     <= auxiliary services for local development 
     keycloak.yaml
     ...         
   entando.json    <= project bundle descriptor
   thumbnail.jpg   <= bundle thumbnail
 ```
+>Note: Platform component names `pageModels` and `contentModels` are deprecated in favor of `pageTemplates` and `contentTemplates`, respectively, on Entando 7.1.
```

Comment:
I added this note but wondering if it's overkill. This is already pointed out on the bundle-components-details page

Reply:
I tend to agree. Maybe link to the bundle-component-details page instead, since that's where the full list of available platform components are listed? That would reduce the duplication and connect the dots.